eng/scripts/verify-npm-tags.ps1

```diff
@@ -25,6 +25,11 @@ Write-Host "Original dist-tag: $parsedOriginalDistTags"
 Write-Host "Current dist-tag: $packageDistTags"
 Write-Host "Intend to add tag $intendedTag to version $intendedTagVersion"
 
+if ($intendedTag -eq "") {
```

Comment:
```suggestion
if ([string]::IsNullOrEmpty($intendedTag)) {
```

Reply:
You can also just do `if (!$intendedTag) {`